src/main/java/lotto/controller/LottoGame.java

```diff
@@ -0,0 +1,26 @@
+package lotto.controller;
+
+import lotto.domain.LottoBuy;
+import lotto.domain.LottoNumber;
+import lotto.domain.LottoTicket;
+import lotto.domain.Money;
+import lotto.domain.WinningLotto;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+public class LottoGame {
+
+    public static void playLotto() {
+        Money money = new Money(InputView.getMoney());
+        LottoBuy lottoBuy = new LottoBuy(money);
+        LottoTicket lottoTicket = lottoBuy.buyAutoLottos(lottoBuy.getCount());
```

Comment:
```suggestion
        LottoTicket lottoTicket = lottoBuy.buyAutoLottos();
```
이렇게 처리할 수 있을 것 같습니다.